src/app.js

```diff
@@ -51,6 +51,8 @@ app.use(
 
 app.use(requestIp.mw());
 
+app.use(morganMiddleWare);
```

Comment:
inject this middleware at the end of all the middlewares. After `app.use(passport.session());` to be precise.